### ISSUE TITLE
Tracking adding writable streams to FF Service Workers

### ISSUE THREAD

**Issue by momack2** [ipfs/in-web-browsers#138]
Passing along feedback from @ianopolous on Peergos - they are using writable streams in service workers, which is enabled in Chrome + others but isn't yet enabled in FireFox. Curious if we're tracking [this issue](https://bugzilla.mozilla.org/show_bug.cgi?id=1128959) and at all engaged with the folks at Mozilla about adding this capability?

**Comment by autonome** [comment]
Readable streams only just got turned on by default in Firefox nightly last
month, which means will ship in 65 (IIRC):
https://twitter.com/dietrich/status/1060542343038885889

Writable streams just got assigned a few days ago, which is always a good
sign: https://bugzilla.mozilla.org/show_bug.cgi?id=1474543

However, I'm not sure that would mean either would be available in SW, so
it would be worth stating your use-case for that on the bug. That way
either it's covered there in the initial implementation, or a separate bug
is filed to track the work and keep on the implementer's radar.

**Comment by momack2** [comment]
Thanks @autonome! 

@ianopolous @cboddy @kevodwyer - can one of you take the lead on describing your use case on the bug here: https://bugzilla.mozilla.org/show_bug.cgi?id=1474543

**Comment by ianopolous** [comment]
Yep, I'm waiting to merge and deploy what I think are the last breaking changes this week, and then I can give a demo url.

**Comment by lidel** [comment]
cc https://github.com/ipfs/js-ipfs-http-client/issues/848
